packages/openid4vc-holder/src/presentations/OpenId4VpHolderService.ts

```diff
@@ -0,0 +1,524 @@
+import type { PresentationSubmission } from './selection'
```

Comment:
Ah I wasn't aware the this implementation of vp. I think if there is some code already there it would be good to add this in a separate PR.